src/particles/wakefields/WakeConvolution.H

```diff
@@ -0,0 +1,33 @@
+#ifndef WAKE_CONVOLUTION_H
+#define WAKE_CONVOLUTION_H
+
+#include "../ImpactXParticleContainer.H"
+#include "../../initialization/InitDistribution.H"
+
+using namespace amrex;
+
+//Constants
+constexpr double e = 1.602e-19;  // Elementary charge [C]
```

Comment:
This can be replaced by using the elementary charge in `ablastr::constant::SI::q_e`